docs/installation.md

```diff
@@ -229,38 +232,28 @@ The following setups describe how to setup the CMSIS-Toolbox using a command lin
+3. Use the menu `View` and open `Source Control`. Select 'Clone Repository' and enter as url [`https://github.com/Open-CMSIS-Pack/vscode-get-started`](https://github.com/Open-CMSIS-Pack/vscode-get-started).
 4. Specify the destination folder to clone to and select 'Open' when asked 'Would you like to open the cloned directory?'
-5. Open the 'Explorer' view (ctrl-shift-e) and select the file 'vcpkg-configuration.json'. This file instructs [Microsoft vcpkg](https://github.com/microsoft/vcpkg-tool#vcpkg-artifacts) to install the prerequisite artifacts required for building the solution.
+5. Use `View` menu 'Explorer' and select the file `vcpkg-configuration.json`. This file instructs [Microsoft vcpkg](https://github.com/microsoft/vcpkg-tool#vcpkg-artifacts) to install the prerequisite artifacts required for building the solution and installs therefore:
 
     - CMSIS-Toolbox
     - cmake 3.25.2
     - ninja 1.10.2
     - arm-none-eabi-gcc 12.2.1-mpacbti (GNU Arm Embedded Toolchain 12.2.1)
 
-6. In case vcpkg shows an error in the VSCode status bar, you can see furth information in the "OUTPUT" for 'vcpkg'.
-In case of 'Error: Unable to resolve dependency ... in <registry>' you may need to update the registry by running 'vcpkg: Run vcpkg command'
-from the 'View' menu's 'Command Palette...' (ctrl+shift+p) typing: `z-ce update <registry>`. 
-7. Open the 'CMSIS' view from the side bar and press the 'Build' button. The last line of the ninja build output will tell you where you can
-find the application elf file. Alternatively you can select 'Build' or                                     'Rebuild' from the context menu of the `*.cprj` file of the solution context
-(e.g. hello.debug+avh.cprj)
-
-
-### Using Visual Studio Code
-
-[Visual Studio Code](https://code.visualstudio.com/) is an effective environment to create CMSIS-based projects.  To setup an environment install the [Keil Studio Pack](https://marketplace.visualstudio.com/items?itemName=Arm.keil-studio-pack) extension from the Visual Studio marketplace.
-
-To work with the **CMSIS-Toolbox** in VS Code use:
-
-- **Terminal - New Terminal** to open a terminal window, on Win64 choose as profile `Command prompt`.
+> **Notes:**
+>
+> - In case vcpkg shows an error in the VSCode status bar, you can see further information in the `vcpkg` output.
+>
+> - In case of `Error: Unable to resolve dependency ... in \<registry\>` you may need to update the registry with the menu `View` option `Command Palette...`, select `vcpkg: Run vcpkg command` and enter: `z-ce update <registry>`.
 
-- In the **Terminal** window, enter the commands for the tools as explained in [project examples](https://github.com/Open-CMSIS-Pack/devtools/blob/main/tools/projmgr/docs/Manual/Overview.md#project-examples).\
+Once the tools are installed you may use the [CMSIS-Toolbox commands](build-tools.md) in a **Terminal** window of VS Code.
```

Comment:
Unfortunately each time vcpkg updates the enviroment the user is required to open a new instance of the terminal to use the updated environment. This is indicated by a Yellow Triangle with exclamation mark next to the shell.